Index: orion.pro
===================================================================
--- orion.pro
+++ orion.pro	2019-03-20 11:27:48.163010494 +0100
@@ -130,10 +130,10 @@
     INSTALLS += target
 
     _desktop.files = distfiles/Orion.desktop
-    _desktop.path = /usr/local/share/applications
+    _desktop.path = /usr/share/applications
 
     _icon.files = distfiles/orion.svg
-    _icon.path = /usr/local/share/icons
+    _icon.path = /usr/share/icons
 
     INSTALLS += _desktop _icon
 }
